ui/app/AppLayouts/Wallet/popups/ReceiveModal.qml

```diff
@@ -24,14 +24,17 @@ StatusModal {
 
     property string address: RootStore.selectedReceiveAccount.address
     property string chainShortNames: ""
+    property var preferredSharingNetworksArray: !!RootStore.selectedReceiveAccount ? RootStore.areTestNetworksEnabled ?
```

Comment:
This seems to be a dupe of `d.preferredSharingNetworksArray`? Get rid of one or the other?

Reply:
aaah good catch was introduced during a rebase, will remove :)